tests/test_config.py

```diff
@@ -628,6 +619,26 @@ def test_process_config_with_wrong_db_url():
     )
 
 
+def test_database_url_no_password():
+    """Test that the database URL can be provided without a password."""
+    expected_url = "postgresql://postgres@localhost:5432/dbostestpy?sslmode=disable"
+    config: DBOSConfig = {
+        "name": "some-app",
+        "database_url": expected_url,
+    }
+    processed_config = translate_dbos_config_to_config_file(config)
+    assert processed_config["name"] == "some-app"
+    assert processed_config["database_url"] == expected_url
+
+    # Make sure we can use it to construct a DBOS Client and connect to the database without a password
```

Comment:
How does this even work? The driver is picking up `PGPASSWORD` out of the environment?

Reply:
Yes, the driver would pick up `PGPASSWORD`. Our tests set this env variable.